extensions.json

```diff
@@ -794,14 +794,14 @@
     },
     {
       "id": "ms-python.python",
-      "download": "https://github.com/Microsoft/vscode-python/releases/download/2020.9.111407/ms-python-release.vsix",
-      "version": "2020.9.111407"
+      "download": "https://github.com/Microsoft/vscode-python/releases/download/2020.9.112786/ms-python-release.vsix",
+      "version": "2020.9.112786"
     },
     {
       "id": "ms-vscode.atom-keybindings",
       "repository": "https://github.com/Microsoft/vscode-atom-keybindings",
-      "version": "3.0.6",
-      "checkout": "v3.0.6"
+      "version": "3.0.9",
+      "checkout": "master"
```

Comment:
The latest release tag from that repo is still `v3.0.6`. Please undo this change.

Reply:
Done!